frontend/.eslintrc.json

```diff
@@ -15,6 +15,14 @@
     "plugin:react/recommended",
     "prettier/@typescript-eslint"
   ],
+  "rules": {
+      "prettier/prettier": [
+          "error",
+          {
+            "endOfLine": "auto"
```

Comment:
How is it breaking? A bit confused from this change as pre-commit we have it set to change to LF ending.

Reply:
I'd get this kind of errors : 

```
$ yarn start
yarn run v1.22.10
$ react-scripts start
(node:21824) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at D:\GitHub\radis-ui\frontend\node_modules\postcss-safe-parser\node_modules\postcss\package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
i ｢wds｣: Project is running at http://192.168.71.1/
i ｢wds｣: webpack output is served from
i ｢wds｣: Content not from webpack is served from D:\GitHub\radis-ui\frontend\public
i ｢wds｣: 404s will fallback to /
Starting the development server...

Failed to compile.

src\constants.ts
  Line 1:36:   Delete `␍`  prettier/prettier
  Line 2:61:   Delete `␍`  prettier/prettier
  Line 3:20:   Delete `␍`  prettier/prettier
  Line 4:25:   Delete `␍`  prettier/prettier
  Line 5:32:   Delete `␍`  prettier/prettier

```

Notice that these files are not files that I edited (I get the crash on a freshly cloned project) so I'm not sure pre-commit would fix anything.

Reply:
oooo, it's coming from the public folder. Since these files are generated, probably we shouldn't be running any linting on them, so that would probably be the best fix here.

Reply:
Hm, weird! I'll take a closer look at this later.